tests/datasets/ushahidi/Base.yml

```diff
@@ -233,76 +233,111 @@ form_stages:
     show_when_published: 1
     task_is_internal_only: 0
     type: "post"
+    priority: 1
   -
     id: 2
     form_id: 1
     label: "2nd step"
     show_when_published: 1
     task_is_internal_only: 0
     type: "task"
+    priority: 2
```

Comment:
ok so this is actually important/good: we are being explicit about priorities to be able to test things more safely, especially the order of how we return things.